src/Components/WebView/WebView/src/Services/WebViewRenderer.cs

```diff
@@ -9,6 +9,7 @@ namespace Microsoft.AspNetCore.Components.WebView.Services;
 
 internal sealed class WebViewRenderer : WebRenderer
 {
+    private static readonly ComponentPlatform _componentPlatform = new("Webview", isInteractive: true);
```

Comment:
Similar suggestion to my other one (was this casing intentional?):
```suggestion
    private static readonly ComponentPlatform _componentPlatform = new("WebView", isInteractive: true);
```

Reply:
Same reasoning as above, if Webassembly is one word, Webview should be too.